sdk/messaging/azservicebus/messageBatch.go

```diff
@@ -34,29 +34,18 @@ func newMessageBatch(maxBytes int) *MessageBatch {
 	return mb
 }
 
-type MessageTooLarge interface {
-	MessageTooLarge()
-}
-
-type errMessageTooLarge struct {
-}
-
-func (e errMessageTooLarge) Error() string {
-	return "message too large to fit in batch"
-}
-
-func (e errMessageTooLarge) NonRetriable()    {}
-func (e errMessageTooLarge) MessageTooLarge() {}
-
 // Add adds a message to the batch if the message will not exceed the max size of the batch
-// If the message is too large, an error of type 'ErrMessageTooLarge' will be returned.
-func (mb *MessageBatch) Add(m *Message) error {
+// This function will return:
+// (true, nil) if the message was added.
+// (false, nil) if the message was too large to fit into the batch.
```

Comment:
If it's just that the batch is full I would reword this "if the batch is full". The current wording makes it read like if the message were smaller it would fit.